### PR TITLE
added tests and test runners

### PR DESCRIPTION
**Pull request by anunagula3**

### CODE REVIEW

File: src/test/java/pages/commonPage.java
Comment (dhirajshettymoj) [inline src/test/java/pages/commonPage.java:1]: can we change the name to CommonPage - java convention.

File: src/test/java/pages/commonPage.java [hunk 1]
```diff
@@ -70,11 +70,27 @@ public void choosePrisoners() {
...
     }
 
+    public void chooseBristol(){
+        click("xpath","//a[contains(@data-test, 'change-establishment')]");
+        selectRadioButton("xpath","//*[@id='BLI']");
+
+    }
+
+
 }
```
Comment (dhirajshettymoj) [inline src/test/java/pages/commonPage.java:96]: blank line at EOF and a few other files below

File: src/test/java/pages/commonPage.java
Comment (anunagula3) [reply src/test/java/pages/commonPage.java:1]: circle ci is caching the class name for some reason. will be looking into apart of another task.

File: src/test/java/pages/commonPage.java (hunk 1 above)
Comment (anunagula3) [reply src/test/java/pages/commonPage.java:94]: address this

**Pull request merged**